IntegrationTests/src/bkr/inttest/labcontroller/test_distro_import.py

```diff
@@ -1522,6 +1522,34 @@ def test_rhel8_import(self):
         trees = self.dry_run_import_trees(['%sRHEL8Alpha/RHT/8.0-Alpha/' % self.distro_url])
         self.assertItemsEqual(trees, [self.x86_64_rhel8])
 
+    def test_preserve_install_options(self):
+        trees = self.dry_run_import_trees([
+            '--preserve-install-options',
+            '%sRHEL8Alpha/RHT/8.0-Alpha/' % self.distro_url])
+        for key in (u'kernel_options', u'kernel_options_post', u'ks_meta'):
+            del self.x86_64_rhel8[key]
+        # Verify that we do not have any of our Install Option values in the dictionary
+        self.assertItemsEqual(trees, [self.x86_64_rhel8])
+
+    def test_preserve_install_options_with_install_opt_fail(self):
+        # import the tree with --preserve-install-options and one of --kopt,
+        # --kopts-post, or --ks-meta. This should fail.
+        for opt_name, key_name in (
+             ('--kopts', u'kernel_options'),
+             ('--kopts-post', u'kernel_options_post'),
+             ('--ks-meta', u'ks_meta')
+        ):
+            try:
+                trees = self.dry_run_import_trees([
```

Comment:
Drop variable local variable `trees`

Reply:
I wonder why line 1513 doesn't have same error?  I copied code from there. Strange.

Reply:
Done

Reply:
Because it is old code and we didn't enforce it. Back in good old times.